### ISSUE TITLE
New Style: Incinerator 

### ISSUE THREAD

**Issue by Nikita74Russian** [BEEmod/BEE2-items#2694]
Just try to remake like on the picture.
I am not asking to FULLY remake the things on the pic.
![20180508025925_1](https://user-images.githubusercontent.com/38806976/39775769-5073e9f2-5318-11e8-80fc-37a6509a56da.jpg)

Goo Pits replaced with fire, incinerators are not needed to remake, just ignore the emergency intelligence incinerator and it's fire.pcf effects. Whole chamber is highlited with orange, some items would malfunction due to the temperature. Buttons would glitch between changing active and unactive colours, broken antlines expected.
Information

Basically overgrown style + P1 + upcoming incinerator skybox.

**Comment by Nikita74Russian** [comment]
The tubes would be a placeable decoration item. Exclusively for this style! I hope idea wont be rejected. I really want this to be added.

**Comment by Nikita74Russian** [comment]
A thumbs down without explaining? Good job Konclan, even though your ideas almost get rejected everytime like me, doesn't mean you can hate the idea itself. It's basically a good idea.

**Comment by HugoBDesigner** [comment]
At least he makes his ideas come to reality

**Comment by Nikita74Russian** [comment]
He tries to. But somehow he does do it. That he disliked my Incinerator Style idea doesn't mean its a bad idea.

**Comment by Nikita74Russian** [comment]
Now, explain. Do you both guys like idea? If no, why not? Explain more. I want your feedbacks so i could make a improvement so the style would be approved as a good idea as my #2646 idea because it was greatly appreciated by @LautaroL20 and TeamSpen, so it may be actually a reality. I hope so. @Teamspen210, notice this issue. I want to know your feedback.

**Comment by Nikita74Russian** [comment]
I am simply trying to improve the Skybox, making it a separate style.. XD!

**Comment by King0191** [comment]
@Nikita74Russian 
Chances are. That these tubes would replace the Vactube in this style. Because exclusive Style items are NOT and never WILL BE added. The members and contributors of BEEmod have made that clear.

**Comment by Nikita74Russian** [comment]
And no, not a duplicate of my own. This style based on Overgrown and P1, unstyled things would use Overgrown or Portal 1 things, (controlled by stylevar.) So i am just wanna make the Bee better but i can't commit my own idea and make @Teamspen210 respect it. It's so freakin hard to reach approvement from owner. Because ik this is a strict community that wants us to use better things, @LautaroL20, @HugoBDesigner, @TeamSpen210. Thanks for making BEE mod. Its the best mod for portal 2 puzzlemaker i ever installed. Thank you. You made my life simpler than it was before! I was struggling to make a hammer map. Especially to @LautaroL20, let's have a peacefull relationships. I want to declare this idiotly war and start a good community. Once i learn hammer, i would do something that you all would appreciate. And it gets added to bee. Thank you for doing this good thing. I really love BeeMod and all of their developers. 

--With appreciation, Nick.

**Comment by Nikita74Russian** [comment]
@King0191 Yeah, i am gonna edit the comment. Vactubes are gonna be replaced with incinerator tubes.

**Comment by Nikita74Russian** [comment]
This is kinda pretty good idea. A test conditions in incinerator room, really gives adrenaline, The neurotoxin timers are replaced with.... Give me guys a idea! Same with vents, i need ideas.

**Comment by Nikita74Russian** [comment]
I will edit the comment since i find a good idea and use it. So people would upvote to this, then it gets commited and may appear in pre-release 30

**Comment by Nikita74Russian** [comment]
I hope it wont be tagged as "Invalid" and "Duplicate." There's a LOT of differences, and this is possible to do.

**Comment by Nikita74Russian** [comment]
Stop disliking every comment i post, it's a thing called: Disrespecting people for no reason.

**Comment by King0191** [comment]
@Nikita74Russian Let me give you some advice here. If you have a lot of things to say. don't make three separate comments. Put them all in the same comment and space it out. it spams the GitHub channel on the Discord and also spams Emails. I would recommend this just for everyone's sanity. and so you don't get banned. because **OCCASIONALLY** You have half-decent ideas

**Comment by Nikita74Russian** [comment]
Give me a tip how to make a decent idea

**Comment by HugoBDesigner** [comment]
Step 1: Acquire a brain
Step 2: Think
Step 3: ???
Step 4: Profit!

**Comment by Nikita74Russian** [comment]
instructions not clear stuck on step 3

**Comment by Nikita74Russian** [comment]
Jk. Because my brain makes ideas that he thinks are good.

**Comment by King0191** [comment]
Best is to read the FAQ and see what it says. And then make sure your idea doesn't mention anything it says to NOT suggest. Also check for your idea in the search bar. so you don't make a duplicate and waste the time of HugoBDesigner or LautaroL20 by making them have to tag your issue and close it.

**Comment by Nikita74Russian** [comment]
Link me the FAQ please. I did not see it.

**Comment by Nikita74Russian** [comment]
I don't remember the link that leads to it.

**Comment by HugoBDesigner** [comment]
Friendly reminder that this isn't a chatting server, so it's best if you edit your messages to include new information instead of posting 2, 3, 4 or more times in a row.
Here's the FAQ: https://github.com/BEEmod/BEE2-items/wiki/FAQ

**Comment by HelloIamaPersonThatIsAliveAndHealthily** [comment]
Congratulations, kid. Your dummy suggestion made my Bee´s Bingo to achieve the eighth checkmark ("Someone suggest a terrible Style"):

![BEE2 BINGO](https://user-images.githubusercontent.com/29079770/39781484-a8da2f58-52e5-11e8-8434-0355df48ab95.png)

Also, nice ammount of thumbs down btw!

**Comment by Konclan** [comment]
Thank

**Comment by HelloIamaPersonThatIsAliveAndHealthily** [comment]
Not to mention that this kid would have easily fill the "A rookie needs help to make Bee2 work..." but for when i make my own revision of the Bee2´s bingo, it was a few days late :(      however i am certainly sure that someother or even this kid will fill it this month, so hopes arent lost after all.

**Comment by LautaroL20** [comment]
Kid, your aphocaliptic suggestion would not be consistent with any of the objects of Bee/P2. The "Style" would be very disruptive for people who only wants to focus on solve their puzzles.

**Comment by Nikita74Russian** [comment]
Same you can say with other styles. Overgrown has plants and stuff and why is it added?

**Comment by Nikita74Russian** [comment]
Stop closing a valid issue.
